src/features/components/atoms/input/checkbox/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import CheckboxInput from '../index'
+
+describe('Test name', () => {
+    test('should be true', () => {
+        render(
+            <CheckboxInput>
+                {' '}
+                <div>shayan</div>
+            </CheckboxInput>,
+        )
+        expect(screen.getByText('shayan')).toBeInTheDocument()
```

Comment:
please add assertions for `checked` and `unchecked` status of the component, thanks :pray :fire:

Reply:
updated the test. would you please check it again?